client.go

```diff
@@ -308,11 +314,21 @@ func (c *TunaSessionClient) getPubAddrsFromRemote(ctx context.Context, remoteAdd
 		return nil, ErrClosed
 	}
 
+	valid := 0
 	for _, addr := range pubAddrs.Addrs {
 		if len(addr.IP) > 0 && addr.Port != 0 {
-			return pubAddrs, nil
+			valid++
 		}
 	}
+	if valid == len(pubAddrs.Addrs) {
```

Comment:
Should always set it because it's the latest response

Reply:
Yes, modified.